pkg/security/probe/process_killer.go

```diff
@@ -24,6 +24,10 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+const (
+	userSpaceKillWithinMillis = 1000
```

Comment:
The PR description says CreateTime has to be within 2 seconds but this is 1 second. Not sure which one is correct.

Reply:
that is -1 +1 sec